app/Http/Controllers/Backend/CategoryController.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests\Backend\CategoryRequest;
+use App\Http\Controllers\Controller;
+use App\Models\Backend\Category;
+use File;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the products.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $cates = Category::paginate(10);
```

Comment:
[rank B] Please avoid use hardcode as less as good. So, you need define a constant for number item per page and use it as global value.